lib/tasks/dataverse_migration.rake

```diff
@@ -54,33 +55,33 @@ namespace :migration do
       MigrationLogger.info "Processing the object #{object_id}"
       #reading the metadata file
       metadata_file = Nokogiri::XML(File.open(file))
-      metadata = metadata_file.xpath("//ddi_to_dcterms",NS)
+      metadata = metadata_file.xpath("//ddi_to_dcterms", MigrationConstants::NS)
```

Comment:
Noticed some weird dependencies in our rake tasks. We are depending on this NS constant that is defined in the migration.rake task. If it is a dependency and needed in other rake tasks, then make it so. Never make the assumption it is just loaded and avaiable across rake tasks. (When testing in isolation this was a big pain figuring this out). This is what the next few files are doing (lots of white space diffs). I pulled the NS constant into a MigrationConstants module and requiring it where needed.